@iamriya Do check there are errors,
Compiler message:
lib/pages/plan_journey_page.dart:2:8: Error: Error when reading 'lib/pages/journey_page.dart': The system cannot find the file specified.

import 'package:metroapp/pages/journey_page.dart';
       ^
lib/pages/plan_journey_page.dart:201:59: Error: Method not found: 'JourneyPage'.
                        MaterialPageRoute(builder: (_) => JourneyPage()));
                                                          ^^^^^^^^^^^
lib/pages/plan_journey_page.dart:201:59: Error: The method 'JourneyPage' isn't defined for the class '_PlanJourneyPageState'.
 - '_PlanJourneyPageState' is from 'package:metroapp/pages/plan_journey_page.dart' ('lib/pages/plan_journey_page.dart').
Try correcting the name to the name of an existing method, or defining a method named 'JourneyPage'.
                        MaterialPageRoute(builder: (_) => JourneyPage()));
                                                          ^^^^^^^^^^^
Compiler failed on D:\metroapp\lib\main.dart

FAILURE: Build failed with an exception.

* Where:
Script 'C:\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 765

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebugX86'.
> Process 'command 'C:\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

--- a/lib/pages/journey_page.dart
+++ b/lib/pages/journey_page.dart
@@ -1,0 +1,20 @@
+import 'package:flutter/material.dart';
+
+class JourneyPage extends StatefulWidget {
+  JourneyPage({Key key}) : super(key: key);
+
+  @override
+  _JourneyPageState createState() => _JourneyPageState();
+}
+
+class _JourneyPageState extends State<JourneyPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+       appBar: AppBar(
+         title: Text('Journey Page'),
+       ),
+       body: Container(),
+    );
+  }
+}

